### PR TITLE
Add missing plugin for image raw

### PR DESCRIPTION
**Pull request by PerrineAguiar**
This PR adds missing plugin for raw images.

### CODE REVIEW

**Comment by mcopejans** [comment]
LGTM. Is it possible to add a warning when some plugins cannot be found?

**Comment by PerrineAguiar** [comment]
@mcopejans Thanks for your review. As you suggested I added a warning when a plugin is not available. I am not sure if it would have prevented this bug but it is still a good thing.

**Comment by mcopejans** [comment]
Throwing an exception would certainly have prevented this bug...

File: TangoRosStreamer/nodelet_manager/src/main/jni/jni.cc [hunk 1]
```diff
@@ -68,6 +70,18 @@ Java_eu_intermodalics_nodelet_1manager_NodeletManager_execute(
     }
     LOG(INFO) << "Finished loading nodelets.";
 
+    // Check that all necessary plugins are available.
+    pluginlib::ClassLoader<image_transport::PublisherPlugin> image_transport_pub_loader("image_transport", "image_transport::PublisherPlugin");
+    if (!image_transport_pub_loader.isClassAvailable("image_transport/raw_pub")) {
+      LOG(WARNING) << "Plugin image_transport/raw_pub is not available.";
+      return 1;
+    } else if (!image_transport_pub_loader.isClassAvailable("image_transport/compressed_pub")) {
```
Comment (mcopejans) [inline TangoRosStreamer/nodelet_manager/src/main/jni/jni.cc]: This should be just `if`, not `else if`.

**Pull request merged**